components/ProgressView/src/MDCProgressView.m

```diff
@@ -292,7 +292,7 @@ + (UIColor *)defaultTrackTintColorForProgressTintColor:(UIColor *)progressTintCo
 
 - (void)updateProgressView {
   // Update progressView with the current progress value.
-  CGFloat progressWidth = MDCCeil(self.progress * CGRectGetWidth(self.bounds));
```

Comment:
Tip: Please use `MDCCeilScaled` which will align the width to pixel values of the screen but keep the "ceiling" behavior.  I believe that aligns with the goal of this PR and will prevent alignment on sub-pixel boundaries, which results in blurrier/less-crisp rendering.

Reply:
Done.